pkg/storage/export/export/export.go

```diff
@@ -768,6 +824,12 @@ func (ctrl *VMExportController) createExporterPodManifest(vmExport *exportv1.Vir
 		Value: currentTime().Add(deadline).Format(time.RFC3339),
 	})
 
+	// If no tokenSecretRef has been specified, we use one specific to the current export object
+	tokenSecretRef := vmExport.Spec.TokenSecretRef
+	if tokenSecretRef == "" {
+		tokenSecretRef = getDefaultTokenSecretName(vmExport)
```

Comment:
I think it would be useful to store the token secret name in the VM export status.  So refer to that in this function instead of calling `getDefaultTokenSecretName` again?